packages/salesforcedx-vscode-apex/src/commands/metadataOrchestrator.ts

```diff
@@ -140,6 +160,7 @@ export class MetadataOrchestrator {
       requests.push(request);
     }
 
+    const response = this.gatherContext(sourceUri);
```

Comment:
@CristiCanizales I was thinking that context gathering would occur after eligibility is determined and the class is eligible for generation.

Reply:
Yes! Right now it's just like a placeholder so we can play with the response when doing prompt engineering